### ISSUE TITLE
Organizational: Due dates for homework

### ISSUE THREAD

**Issue by itsthefedora** [Statistical-Connectomics-Sp15/intro#100]
... Do we have them?

**Comment by sgomezr** [comment]
yup.. do we have them? Also, are they counted per week or per class?

**Comment by mrjiaruiwang** [comment]
I was under the assumption that they were due a week after, but I also heard, to quote from an email from Greg, "… the questions are completely optional to participate in that week - do so at your own leisure." So I don't know if "leisure" means freedom of option or time period.

**Comment by gkiar** [comment]
Sometimes they matter, sometimes they don't. :)
... To be not totally unhelpful: if you're presenting a paper, or we're talking about a certain algorithm and ask you to have tested it first, etc., it is obviously helpful if you do them in a timely manner. For "marking" purposes, if you do them, you get credit. That means, the last day of the class will be the last day I count submissions of things.

Jerry takes what I said a bit out of context, but I'll clarify:
Homeworks, each lecture, are to participate in 4 questions. If stated otherwise (i.e. we say, "write up a statistical decision theoretic" or "test kmeans with the intent of making it fail"), then this task replaces the questions for that lecture.

tl;dr: ask questions, do work, put things you do on the repo, be happy.
